Mathlib/Analysis/Complex/CauchyIntegral.lean

```diff
@@ -579,7 +579,7 @@ theorem _root_.DifferentiableOn.analyticOn {s : Set ℂ} {f : ℂ → E} (hd : D
 differentiable on `s`. -/
 protected theorem _root_.DifferentiableOn.contDiffOn {s : Set ℂ} {f : ℂ → E} {n : ℕ}
     (hd : DifferentiableOn ℂ f s) (hs : IsOpen s) : ContDiffOn ℂ n f s :=
-  (hd.analyticOnNhd hs).contDiffOn
+  (hd.analyticOnNhd hs).contDiffOn hs.uniqueDiffOn
```

Comment:
Why `AnalyticOnNhd.contDiffOn` needs `UniqueDiffOn` now?

Reply:
The assumptions in `AnalyticOnNhd.contDiffOn` has changed: in the previous version, it was requiring a complete space, but no unique differentiability. While the new version drops completeness, but requires unique differentiability.

Reply:
Does it make sense to have the old version too? Anyway, could you please mention API changes like this one (i.e., not just `ENat -> WithTop ENat`) in the commit message? Also, am I right that we can upgrade this lemma from `n : Nat` to `n : WithTop ENat` for free?

Reply:
Sure, I've added back the old version, under the name `AnalyticOnNhd.contDiffOn_of_completeSpace` (and I've used it in `CauchyIntegral.lean`).

I think this lemma already use `n : WithTop ENat`.